### PR TITLE
Add font-beattingvile v1.0

### PR DESCRIPTION
**Pull request by mcdado**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-fonts/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/font-beattingvile.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'font-beattingvile' do
+  version '1.0'
+  sha256 '0aac26846be14641f46948acd17e3714354699f0f48656de6fca3df1da2bc313'
+
+  # freetypography.com was verified as official when first introduced to the cask
+  url 'https://freetypography.com/freetypo/Beattingvile.zip'
+  name 'Beattingvile'
+  homepage 'https://www.behance.net/rismanginarwan'
```
Comment (vitorgalvao) [inline Casks/font-beattingvile.rb]: Link to the typography’s page, not the user’s.

Comment (mcdado) [reply Casks/font-beattingvile.rb]: Done

File: Casks/font-beattingvile.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
   # freetypography.com was verified as official when first introduced to the cask
   url 'https://freetypography.com/freetypo/Beattingvile.zip'
   name 'Beattingvile'
-  homepage 'https://www.behance.net/rismanginarwan'
+  homepage 'https://garisman.com/'
```
Comment (vitorgalvao) [inline Casks/font-beattingvile.rb]: This still does not link to the font’s page.

Comment (vitorgalvao) [reply Casks/font-beattingvile.rb]: I mean the typeface’s page. My mistake there.

File: Casks/font-beattingvile.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'font-beattingvile' do
+  version '1.0'
+  sha256 '0aac26846be14641f46948acd17e3714354699f0f48656de6fca3df1da2bc313'
+
+  # freetypography.com was verified as official when first introduced to the cask
+  url 'https://freetypography.com/freetypo/Beattingvile.zip'
+  name 'Beattingvile'
+  homepage 'https://garisman.com/'
```
Comment (vitorgalvao) [inline Casks/font-beattingvile.rb]: Link to the typeface’s page.

Comment (mcdado) [reply Casks/font-beattingvile.rb]: Closest link I could find: https://garisman.com/freebies/

**Comment by vitorgalvao** [comment]
Unfortunately, looking at it further, we won’t be able to include this. It’s a [walled build](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) so [which we do not accept](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds).

**Pull request closed without merging**